src/Controllers/Controller.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Masterclass\Controllers;
+
+
+use PDO;
+
+/**
+ * Class Controller
+ * @package Masterclass\Controllers
+ */
+class Controller
```

Comment:
This constructor goes away in lesson 2
On 15 Mar 2016 07:30, "Brandon Savage" <notifications@github.com> wrote:

> In src/Controllers/Controller.php
> <https://github.com/SeanJA/masterclass-repo/pull/1#discussion_r56148667>:
>
> > @@ -0,0 +1,30 @@
> > +<?php
> > +
> > +namespace Masterclass\Controllers;
> > +
> > +
> > +use PDO;
> > +
> > +/**
> > + * Class Controller
> > + * @package Masterclass\Controllers
> > + */
> > +class Controller
>
> Here be dragons.
>
> Creating a base controller can seem like it's going to save you a lot of
> work, but it often doesn't. You're defining a constructor that you're going
> to have to live with forever, meanwhile you may have objects that you don't
> need/want that are always required.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/SeanJA/masterclass-repo/pull/1/files#r56148667
>